src/lib/decorators.ts

```diff
@@ -25,7 +25,7 @@ import {LitElement} from '../lit-element.js';
 import {PropertyDeclaration, UpdatingElement} from './updating-element.js';
 
 export type Constructor<T> = {
-  new (...args: unknown[]): T
+  new (...args: any[]): T
```

Comment:
Add a lint ignore comment, and if this isn't a lint warning, we should make it one